### ISSUE TITLE
Loading main.jsx

### ISSUE THREAD

**Issue by GiladShoham** [sysgears/apollo-fullstack-starter-kit#21]
Hi,

I saw that client/main.jsx is used only by client/index.jsx.
index.jsx is defined as the entry point of clientConfig in webpack.config.js.

I have tried to just log some message in the index.jsx / main.jsx and i didn't see the log.
Also look on the build folder and didn't saw the file under build/client (i only see there assets.json).

I wonder how those filed are used?

I use apollo-passport and tried to configure it inside main.jsx (because of apollo-passport use localStorage) so I need to configure it only on the client. 

But I never get into this code.

Is this code run only after build and not on regular dev with an npm start?
If yes, how can I make it usable for testing purposes? 

Thanks!
Gilad.

**Comment by vlasenko** [comment]
@GiladShoham
> I have tried to just log some message in the index.jsx / main.jsx and i didn't see the log.

Yes, you should see the log in the web browser JavaScript console if you have logging statements in client/index.jsx or main.jsx. 
> Also look on the build folder and didn't saw the file under build/client (i only see there assets.json).
During development most client-side files are generated into memory only, this is how webpack dev server works, it's normal. 

> Is this code run only after build and not on regular dev with an npm start?
All the code run during `npm start` and is available for testing purposes. The build phase is needed only for final delivery of application to production.

**Comment by GiladShoham** [comment]
@vlasenko 
Wow! that was quick!
Thanks!

Ok I see it now in the browser. 
Why is this not logged in to the console as other logs from the frontend?

I just can't make the client configuration of apollo-passport to work properly..
Will keep trying.

Thanks!

**Comment by vlasenko** [comment]
@GiladShoham Front end is not logging to server logs. Only webpack-for-frontend is logging to server logs. All the front end is logging to web browser console only.

**Comment by vlasenko** [comment]
@GiladShoham There is a notice on apollo-passport package, that it is deprecated and link to https://github.com/js-accounts/accounts. But `js-accounts` has another notice, that it is not ready yet for consumption... So how are you trying to work with them, if they are not ready yet?

**Comment by GiladShoham** [comment]
@vlasenko 
I'm working with apollo-passport for now because js-accounts is not ready yet.
I've opened a question to the author [here](https://github.com/apollo-passport/apollo-passport/issues/11), he says that there is no problem to work with apollo-passport meantime.
And the transition to js-account in the future should be easy.

By the way, I have just succeeded to make it kind of work:)

p.s - is your email in GitHub are updated? I want to introduce you new repos i just started which handle graphql/apollo errors in a much better way.
I would glad to discuss it and get feedback from you. (just started yesterday, so it's a very early stage, but works great with my project)
